forecasting/src/autogluon/forecasting/models/gluonts/models.py

```diff
@@ -0,0 +1,172 @@
+        The number of channels (i.e. filters or convolutions) for each layer of the HierarchicalCausalConv1DEncoder.
+        More channels usually correspond to better performance and larger network size.
+        (default: [30, 30, 30])
+    dilation_seq: List[int]
+        The dilation of the convolutions in each layer of the HierarchicalCausalConv1DEncoder.
+        Greater numbers correspond to a greater receptive field of the network, which is usually
+        better with longer context_length. (Same length as channels_seq) (default: [1, 3, 5])
+    kernel_size_seq: List[int]
+        The kernel sizes (i.e. window size) of the convolutions in each layer of the HierarchicalCausalConv1DEncoder.
+        (Same length as channels_seq) (default: [7, 3, 3])
+    use_residual: bool
+        Whether the hierarchical encoder should additionally pass the unaltered
+        past target to the decoder. (default: True)
+    quantiles: List[float]
+        The list of quantiles that will be optimized for, and predicted by, the model.
+        Optimizing for more quantiles than are of direct interest to you can result
+        in improved performance due to a regularizing effect.
+        (default: [0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9])
+    distr_output: gluonts.mx.DistributionOutput()
+        DistributionOutput to use. Only one between `quantile` and `distr_output`
+        can be set. (Default: None)
+    scaling: bool
+        Whether to automatically scale the target values. (default: False if quantile_output is used, True otherwise)
+    """
+
+    def _get_estimator(self):
+        if get_mxnet_context() != mx.context.cpu():
+            self.params["hybridize"] = False
+
+        with warning_filter():
+            return MQCNNEstimator.from_hyperparameters(
+                **self._get_estimator_init_args()
+            )
+
+
+class SimpleFeedForwardModel(AbstractGluonTSModel):
+    """SimpleFeedForward model, i.e. a simple multilayer perceptron for
+     probabilistic forecasts, from GluonTS.
+
+    See `AbstractGluonTSModel` for common parameters.
+
+    Other Parameters
+    ----------------
+    num_hidden_dimensions: int
+        Number of hidden nodes in each layer (default: [40, 40])
+    context_length: int
+        Number of time units that condition the predictions
+        (default: None, in which case context_length = prediction_length)
+    distr_output: gluonts.mx.DistributionOutput
+        Distribution to fit (default: StudentTOutput())
+    batch_normalization: bool
+        Whether to use batch normalization (default: False)
+    mean_scaling: bool
+        Scale the network input by the data mean and the network output by
+        its inverse (default: True)
+    """
+
+    def _get_estimator(self):
+        with warning_filter():
+            return SimpleFeedForwardEstimator.from_hyperparameters(
+                **self._get_estimator_init_args()
+            )
+
+
+class AutoTabularModel(AbstractGluonTSModel):
```

Comment:
I've seen this same idea before in GluonCV: They added `gluoncv.auto` which imported AutoGluon. Then in `autogluon.vision` we imported `gluoncv.auto` which imported `autogluon.core`. We should be extremely careful with this, as it creates very brittle code.

For example: 
```
VALID | AutoGluon = v1
VALID | GluonTS = v1, GluonTS.TabularEstimator depends on AutoGluon v1
ERROR | Release AutoGluon v2 with Forecasting support, AutoGluon v2 depends on GluonTS v1 which depends on AutoGluon v1 -> ag.forecasting requires ag.core=v2, but gluonts.auto requires ag.core=v1.
VALID | Release GluonTS v2, depends on AutoGluon v2>=,<3
VALID | Release AutoGluon v2.1, depends on GluonTS v2
```

We may want to strongly consider moving whatever logic is in `TabularEstimator` to AutoGluon to avoid this dependency version issue. This is what we eventually did for GluonCV and it solved many problems.